### PR TITLE
fix(client): remove enumerable on OR

### LINKED ISSUE #4831: OR operator behaving like AND
## Bug description

Generated query with `OR` uses `AND` instead.

## How to reproduce

With any schema, use query with `OR` in ˙where˙.

```ts
const carriersToProcess = await prisma.carrier.findMany({
	select: {
		carrierID: true,
		lat: true,
		lng: true,
	},
	where: {
		lat: { gt: 0 },
		lng: { gt: 0 },
		origin: { not: { equals: 'S' } },
		OR: {
			regionID: { equals: null },
			districtID: { equals: null },
		},
	},
})
```

## Actual behavior

```sql
SELECT `db`.`carrier`.`carrierID`, `db`.`carrier`.`lat`, `db`.`carrier`.`lng` FROM `db`.`carrier` 
WHERE (
`db`.`carrier`.`lat` > ? AND 
`db`.`carrier`.`lng` > ? AND 
`db`.`carrier`.`Zdroj` <> ? AND (
  `db`.`carrier`.`regionID` IS NULL AND 
  `db`.`carrier`.`districtID` IS NULL )
) 
ORDER BY `db`.`carrier`.`carrierID` ASC LIMIT ? OFFSET ?
```

## Expected behavior

```sql
SELECT `db`.`carrier`.`carrierID`, `db`.`carrier`.`lat`, `db`.`carrier`.`lng` FROM `db`.`carrier` 
WHERE (
`db`.`carrier`.`lat` > ? AND 
`db`.`carrier`.`lng` > ? AND 
`db`.`carrier`.`Zdroj` <> ? AND (
  `db`.`carrier`.`regionID` IS NULL OR 
  `db`.`carrier`.`districtID` IS NULL )
) 
ORDER BY `db`.`carrier`.`carrierID` ASC LIMIT ? OFFSET ?
```

## Environment & setup

- OS: Windows
- Database: MySQL
- Node.js version: v12.16.1
- Prisma version:

```
@prisma/cli          : 2.12.1
@prisma/client       : 2.12.1
Current platform     : windows
Query Engine         : query-engine cf0680a1bfe8d5e743dc659cc7f08009f9587d58 (at node_modules\@prisma\engines\query-engine-windows.exe)
Migration Engine     : migration-engine-cli cf0680a1bfe8d5e743dc659cc7f08009f9587d58 (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine : introspection-core cf0680a1bfe8d5e743dc659cc7f08009f9587d58 (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary        : prisma-fmt cf0680a1bfe8d5e743dc659cc7f08009f9587d58 (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Studio               : 0.322.0
```

### PR DESCRIPTION
**Pull request by williamluke4**
fixes #4831

### CODE REVIEW

File: src/packages/client/src/generation/TSClient/Input.ts [hunk 1]
```diff
@@ -159,7 +159,11 @@ export class InputType implements Generatable {
 ${indent(
   fields
     .map((arg) =>
-      new InputField(arg /*, type.atLeastOne && !type.atMostOne*/).toTS(),
+      new InputField(
+        arg /*, type.atLeastOne && !type.atMostOne*/,
```
Comment (williamluke4) [inline src/packages/client/src/generation/TSClient/Input.ts:163]: @timsuchanek Can I remove this comment?

Comment (timsuchanek) [reply src/packages/client/src/generation/TSClient/Input.ts:163]: Yes

**Comment by timsuchanek** [comment]
Please add runtime and type tests :)

File: src/packages/client/src/runtime/getPrismaClient.ts [hunk 2]
```diff
@@ -990,6 +990,12 @@ new PrismaClient({
           `executeRaw and queryRaw can't be executed on a model basis. The model ${model} has been provided`,
         )
       }
+      if (args?.where && args.where['OR'] && !Array.isArray(args.where['OR'])) {
```
Comment (timsuchanek) [inline src/packages/client/src/runtime/getPrismaClient.ts:993]:
```suggestion
      if (args?.where?.['OR'] && !Array.isArray(args.where['OR'])) {
```

**Pull request closed without merging**